xbmc/cores/VideoPlayer/Process/VideoBuffer.cpp

```diff
@@ -318,14 +318,21 @@ void CVideoBufferPoolSysMem::Return(int id)
   m_free.push_back(id);
 }
 
-void CVideoBufferPoolSysMem::Configure(AVPixelFormat format, int width, int height)
+void CVideoBufferPoolSysMem::Configure(AVPixelFormat format, int width, int height, int aligned_width, int aligned_height, int size)
```

Comment:
@peak3d - is it possible to provide width/height/stride at point Get is called (CAddonVideoCodec::GetFrameBuffer)?